articles/appliance/admin/rate-limiting.md

```diff
@@ -0,0 +1,67 @@
+---
+name: Rate Limiting in the Appliance
+description: How to enable, configure, and test for rate limiting in the Appliance
+---
+# Appliance: Rate Limiting
```

Comment:
Aren't we supposed to use the `Private SaaS (PSaaS) Appliance` thing when we refer to appliance?